src/core/main.c

```diff
@@ -141,9 +141,9 @@ static usec_t arg_runtime_watchdog;
 static usec_t arg_reboot_watchdog;
 static usec_t arg_kexec_watchdog;
 static usec_t arg_pretimeout_watchdog;
-static char *arg_early_core_pattern;
+static char *arg_early_core_pattern = NULL;
 static char *arg_watchdog_pretimeout_governor;
-static char *arg_watchdog_device;
+static char *arg_watchdog_device = NULL;
```

Comment:
Please drop this chunk: see the comment above in line 117.

Reply:
static variables are initialized to 0. So initializing them to e.g. NULL is a cosmetic thing to make it clearer to the reader what the initialization value is. But here we are initializing them explicitly in a different place, so initialization was removed here.